packages/access-client/src/agent.js

```diff
@@ -535,46 +515,7 @@ export class Agent {
 
     // claim delegations here because we will need an ucan/attest from the service to
     // pair with the session delegation we just claimed to make it work
-    await this.claimDelegations()
-  }
-
-  async claimDelegations() {
```

Comment:
note: removing `claimDelegations` instance method in favor of `claimDelegations(agent: AccessAgent)` use case fn

Reply:
I want to do this for:
* `addProvider`
* `delegateSpaceAccessToAccount`
* (tbh) `authorize`